api/api/utils/cache_control/__init__.py

```diff
@@ -0,0 +1,26 @@
+from functools import wraps
+
+from api.utils.cache_control.backports import cache_control, never_cache  # noqa: E501
+
+
+def async_method_cache_control(**cache_control_kwargs):
+    """
+    Decorate an async method with `cache_control`.
+
+    Only necessary for async view methods because Django's `method_decorator`
```

Comment:
Should we add something like `TODO: Remove when migrating to Django 5, see https://github.com/WordPress/openverse/issues/4015`?